lib/actions/reviewer-survey/index.js

```diff
@@ -0,0 +1,17 @@
+import { laboratoriaAPIAction } from '../helpers';
+
+export const getLastVersion = () => laboratoriaAPIAction({
+  type: 'LAST_VERSION',
+  url: '/reviewer-survey',
+  method: 'GET',
+  key: 'last-version',
+});
+
+
+export const getQuestions = ({ version, ...rest }) => laboratoriaAPIAction({
```

Comment:
This method is meant to **get a reviewer survey**, not just its questions, so `getReviewerSurvey` should be a more appropriate name.